app/src/main/java/com/example/polina/socialnetwork/ServerAPI.java

```diff
@@ -35,26 +35,15 @@
     private String userInfoPath = HOST + "/user/me";
     private String postPath = HOST + "/post";
     private String postGetPath = HOST + "/timeline/";
-
-
-
+    private String postLoadPostSize = "?limit=";
```

Comment:
it's much better to use formated string here: 
For ex.: String.format("?limit=%d&before=%s", ...)